docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,591 @@
+
+* *SQL DB Database View*. Detailed info about each SQL database, including wait time, errors, DTU and storage utilization, size, and read and write input/output.
+
+* *SQL DB Queries*. Info about SQL database queries, including DTU Utilization, errors, and query duration and wait time. 
+
+* *User Activity*. Info about system users, their activity, and requests.
+
+You can use the dashboards they are, or tailor them to meet your needs.
+
+==== Azure_event_hubs plugin
+
+The Azure module uses the `azure_event_hubs` plugin. Basic understanding of the
+plugin is helpful when you set up the Azure module. See
+{logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs plugin
+documentation] for more information about configurations and options. 
+
+
+[[azure-prereqs]]
+==== Prerequisites
+
+These instructions assume that Logstash, Elasticsearch, and Kibana are
+installed and running. The products are
+https://www.elastic.co/downloads[available to download] and easy to install. 
+
+The Elastic Stack 6.4 (or later) is required for this module. 
+
+NOTE: Logstash, Elasticsearch, and Kibana must run locally. You can also run 
+Elasticsearch, Kibana and Logstash on separate hosts to consume data from Azure.
+
+
+[[azure-setup]]
+==== Installation and setup
+
+To get started with the Azure module:
+
+ . Install the {logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs
+ plugin]. 
+ . Set up the Azure module. 
+
+[[azure-plugin-setup]]
+===== Install the plugin
+
+TBD: From the LS directory?
+Run this command to install the plugin:
+
+["source","shell"]
+-----
+bin/logstash-plugin install logstash_input_azure_event_hubs
+-----
+
+[[azure-module-setup]]
+===== Set up the module
+
+TBD: From the LS directory?
+
+Modify this command for your environment, and run it.
+
+["source","shell",subs="attributes"]
+-----
+bin/logstash --modules azure --setup  \
+  -M "azure.var.elasticsearch.username={username}"  \
+  -M "azure.var.elasticsearch.password={pwd}"  \
+  -M "azure.var.kibana.username={username}"  \
+  -M "azure.var.kibana.password={pwd}"  \
+  -M "azure.var.elasticsearch.hosts={hostname}"  \
+  -M "azure.var.kibana.host={hostname}"
+-----
+
+The `--modules azure` option starts a Logstash pipeline for ingestion from Azure
+Event Hubs. The `--setup` option creates an `azure-*` index pattern in
+Elasticsearch and imports Kibana dashboards and visualizations. 
+
+NOTE: The `--setup` option is intended only for first-time setup. If you include `setup` on subsequent runs, your existing Kibana dashboards will be overwritten.
+
+
+[[configuring-azure]]
+==== Configure the module
+
+You can specify <<azure_config_options, options>> for the Logstash Azure module in the
+`logstash.yml` configuration file or with overrides through the command line. 
+
+The azure_event_hubs plugin and the Azure module support two configuration
```

Comment:
there are really 3 configuration modes, command line, basic, and advanced. 

command line = 1 event hub
basic = multiple event hubs same config
advanced = multiple event hubs different config

I think it makes sense to document it this way instead of the 2 ways with the caveat that basic has two ways (command line vs. yml) and the command line only supports a single event hub.